actions/setup.go

```diff
@@ -62,8 +62,11 @@ func DefaultLocalSetup(
 	if err != nil {
 		return nil, err
 	}
-	if err := contracts.AwaitMining(blockchainClient); err != nil {
-		return nil, err
+	if network.Config().Name == "Ethereum Geth reorg" {
```

Comment:
We need to never match on string names of networks. Add this in as a config flag per network, something like:
`await_mining_on_setup: true`